udata/core/spatial/models.py

```diff
@@ -24,6 +24,9 @@
 
 ADMIN_LEVEL_MIN = 1
 ADMIN_LEVEL_MAX = 110
+# Max known population for a zone
+# World population is 6772425850.
+POPULATION_MAX = 2147483647
```

Comment:
OK, so not a problem here because the max computed value is far from it. 
I changed for the world population (approximative) so the number means something and the algo works on any zone